extra-network/syncthing/autobuild/build

```diff
@@ -1,30 +1,65 @@
-install -d "src/github.com/syncthing"
-mv syncthing "src/github.com/syncthing/syncthing"
+# Adapted from Arch Linux
+abinfo 'Preparing build environment ...'
+install -dv "$SRCDIR"/src/github.com/syncthing
```

Comment:
Why use `install` here?

Reply:
Here I just added verbosity based on Arch's PKGBUILD script, should I replace it with `mkdir -pv`?